outputs.tf

```diff
@@ -0,0 +1,6 @@
+output "webapp_url" {
+ value = "https://${azurerm_app_service.main.default_site_hostname}"
+}
+output "travis_webhook" {
+ value = "https://${azurerm_app_service.main.site_credential[0].username}:${azurerm_app_service.main.site_credential[0].password}@${azurerm_app_service.main.name}.scm.azurewebsites.net/docker/hook"
```

Comment:
You need sensitive = true to avoid exposing this sensitive value in your Travis logs.